Add Aspose.PDF for .NET 24.11 release notes (#381)
Aspose/releases.aspose.com: content/en/pdf/net/release-notes/2024/aspose-pdf-for-net-24-11-release-notes.md

This page contains release notes information for Aspose.PDF for .NET 24.11.

{{% /alert %}}

## Improvements and Changes

|**Key**|**Summary**|**Category**|
| :- | :- | :- |
|PDFNET-49833|Automatically update page numbers in the header or footer of a PDF when inserting new pages|Feature|
|PDFNET-58201|Enable RGB to CMYK and CMYK to RGB conversions using a custom ICC profile|Feature|
|PDFNET-58345|Ability to specify a hashing algorithm when creating digital signatures|Feature|
|PDFNET-41994|Support signing documents using the SHA256 hashing algorithm|Enhancement|
|PDFNET-48541|Allow setting the hashing algorithm for external signatures|Enhancement|
|PDFNET-50003|Support 192-bit ECDSA private keys for PKCS7Detached signatures|Enhancement|
|PDFNET-57817|Add support for parsing CMap subtable to replace incorrect ToUnicode tables during HTML export|Enhancement|
|PDFNET-58484|Enable compression without quality loss for 2 BPP and 4 BPP images|Enhancement|
|PDFNET-56893|Cannot correctly check permissions on the PDF file|Bug|
|PDFNET-58429|PDF to PDF/A-3b: the output is not compliant due to widths error|Bug|
|PDFNET-46498|Cannot open 256bit AES encoded PDF due to unhandled exception|Bug|

> |PDFNET-49064|VerifySigned returns an unexpected result|Bug|

Reviewer: ```suggestion
|PDFNET-49064|Method VerifySigned returns an unexpected result|Bug|
```

|PDFNET-58388|Text is always underlined on the page if auto-wrapped|Bug|
|PDFNET-45238|Garbled text during PDF to image rendering|Bug|
|PDFNET-45975|Exception converting to PDF/A|Bug|
|PDFNET-46070|Inconsistent coordinate handling in image and text insertion|Bug|
|PDFNET-44519|PDF to PDF/A-3u - Key not found error occurred|Bug|
|PDFNET-58311|Concurrency error during document conversion|Bug|
|PDFNET-58208|Japanese Characters Garbled When Converting PDF to JPG on Amazon Linux 2023|Bug|
|PDFNET-58451|Content moved to the bottom of the PDF after untagging the table in 508-compliant PDF|Bug|
|PDFNET-51710|Latex to PDF - output is not correct|Bug|
|PDFNET-58190|XPS to PDF: Additional characters are added for specific font|Bug|
|PDFNET-58307|PdfPageStamp misplace stamp in output PDF|Bug|
|PDFNET-58448|Regression: Redacting data from PDF multiplies and shifts the redaction box|Bug|
|PDFNET-39971|Problem opening encrypted document|Bug|
|PDFNET-58236|PdfBookmarkEditor.ExtractBookmark is never completed for certain documents|Bug|
|PDFNET-58439|PDF to XLSX: System.InvalidCastException on Alpine Linux|Bug|
|PDFNET-58216|"Wrong object format" in PDF to PDF/A-1b conversion|Bug|

> |PDFNET-53049|Investigate Exception during getting property TtfFont.Metrics|Bug|

Reviewer: ```suggestion
|PDFNET-53049|An exception is thrown while getting property TtfFont.Metrics|Bug|
```
